src/test/java/de/hpi/bpt/argos/eventProcessing/EventReceiverTest.java

```diff
@@ -77,6 +77,11 @@ public void testReceiveEvent() throws Exception {
 
 		List<String> webSocketMessages = webSocket.awaitMessages(1, 1000);
 		ArgosTestUtil.assertWebSocketMessage(webSocketMessages.get(0), PersistenceArtifactUpdateType.CREATE, "Event");
+
+		JsonObject message = jsonParser.parse(webSocketMessages.get(0)).getAsJsonArray().get(0).getAsJsonObject();
```

Comment:
This method asserts two messages. One for a created event and one for an entity update. In this case, the status of the entity should not be updated and therefore only one message is to be asserted.